sapphire/tests/transformations/test_celestial.py

```diff
@@ -194,6 +202,12 @@ class AstropyEquatorialTests(unittest.TestCase):
     coordinates from astropy.
     """
     def test_pyephem_htoea(self):
+        """
+        Check  celestial.horizontal_to_equatorial_astropy
+        """
+        if not has_astropy:
```

Comment:
There is a nice decorator for this. `@unittest.skipUnless()`

See also: https://github.com/HiSPARC/sapphire/blob/master/sapphire/tests/corsika/test_blocks.py#L133